### PR TITLE
RDK-35331: RDKShell - Key Repeat Updates

### PR DESCRIPTION
**Pull request by mateusz-hobgarski-red**

### CODE REVIEW

File: RDKShell/RDKShell.cpp [hunk 1]
```diff
@@ -5343,6 +5345,62 @@ namespace WPEFramework {
...
+            }
+            else
+            {
+                response["message"] = "please specify initialDelay parameter";
+                returnResponse(false);
+            }
+
+            if (parameters.HasLabel("repeatInterval"))
+            {
+                enabled = parameters["repeatInterval"].Number();
```
Comment (mfiess) [inline RDKShell/RDKShell.cpp:5390]: I believe "enabled" here should be "repeatInterval"

File: RDKShell/RDKShell.cpp [hunk 2]
```diff
@@ -5343,6 +5345,62 @@ namespace WPEFramework {
...
+            }
+            else
+            {
+                response["message"] = "please specify enabled parameter";
+                returnResponse(false);
+            }
+
+            if (parameters.HasLabel("initialDelay"))
+            {
+                enabled = parameters["initialDelay"].Number();
```
Comment (mfiess) [inline RDKShell/RDKShell.cpp:5380]: I believe "enabled" here should be "initialDelay"

**Review by mfiess — CHANGES_REQUESTED** [review]

File: RDKShell/RDKShell.cpp (hunk 1 above)
Comment (mateusz-hobgarski-red) [reply RDKShell/RDKShell.cpp]: corrected

File: RDKShell/RDKShell.cpp (hunk 2 above)
Comment (mateusz-hobgarski-red) [reply RDKShell/RDKShell.cpp]: corrected

**Comment by Firdose2109** [comment]
Can one of the admins verify this patch?

**Comment by anand-ky** [comment]
Closing older PRs. Pls resubmit to latest 2205 sprint.

**Pull request closed without merging**